### DISCUSSION TITLE
How to disable autosave in TextInputColumn?

### DISCUSSION THREAD

**Discussion by faridsa** [filamentphp/filament#6187] [Help]
Hi, I am using `TextInputColumn` on a table to display editable fields. The point is that those fields have an autosave feature when the field loses focus. I understand that there must be a way to disable it, but I can't figure out what that way is.
Thanks in advance!

**Comment by danharrin** [comment]
There is no button to save, so that is the only way at the moment

**Reply by faridsa** [reply]
Oh! I have created a save button using the following code 
```
return $table
    ->actions([
                Action::make('save')
                    ->action(fn($record) => $record->save())
                    ->label('Save')
                    ->icon('heroicon-o-check')
            ])
```

So, it is useless, right?

**Reply by danharrin** [reply]
Yeah that won't work

**Reply by juanborras** [reply]
Any advanced? some ->autoSave(false) or something like that?
